crates/server/src/api/routes/creators/get_quests_by_creator_id.rs

```diff
@@ -49,7 +49,7 @@ pub async fn get_quests_by_creator_id(
 
     match db
         .get_quests_by_creator_id(
-            &user_address,
+            &user_address.to_lowercase(),
```

Comment:
is this needed if we're lowercasing it inside `get_user_address_from_request`?

Reply:
yes, `user_address` cames from the request and not from `get_user_address_from_request`